modules/imgproc/src/fixedpoint.inl.hpp

```diff
@@ -261,19 +268,23 @@ class ufixedpoint32
     CV_ALWAYS_INLINE bool isZero() { return val == 0; }
     static CV_ALWAYS_INLINE ufixedpoint32 zero() { return ufixedpoint32(); }
     static CV_ALWAYS_INLINE ufixedpoint32 one() { return ufixedpoint32((1U << fixedShift)); }
+
+    static CV_ALWAYS_INLINE ufixedpoint32 fromRaw(uint32_t v) { return ufixedpoint32(v); }
+    CV_ALWAYS_INLINE ufixedpoint32 raw() { return val; }
```

Comment:
Could you please update return type here and for `CV_ALWAYS_INLINE ufixedpoint16 raw()` as well